src/components/player-chips.tsx

```diff
@@ -15,38 +23,128 @@ const StyledDot = styled(Dot)({
   borderWidth: 2,
 });
 
-export function PlayerChips({ fill }: { fill?: boolean }) {
+export function PlayerBlueChip({
```

Comment:
These should all be a single component. `PlayerChip` which selects their bgColor value based on the color prop